operations/mimir-mixin/dashboards/writes.libsonnet

```diff
@@ -137,6 +137,47 @@ local filename = 'mimir-writes.json';
         )
       )
     )
+    .addRowIf(
+      $._config.forwarding_enabled,
+      $.row('Distributor Forwarding')
+      .addPanel(
+        $.panel('Requests / sec') +
+        $.successFailurePanel(
+          |||
+            sum(  
+              rate(
+                cortex_distributor_forward_requests_total{%(distributorMatcher)s}[$__rate_interval]
+              )
+            )
+            - 
+            sum(
+              rate(
+                cortex_distributor_forward_errors_total{%(distributorMatcher)s}[$__rate_interval]
+              )
+            )
+          ||| % {
+            distributorMatcher: $.jobMatcher($._config.job_names.distributor),
+          },
+          'sum by (status_code) (rate(cortex_distributor_forward_errors_total{%(distributorMatcher)s}[$__rate_interval]))' % {
```

Comment:
I think this won't work with `successFailurePanel()`, because it sets the legend to `failure` while here you want to have the legend set to `{{ status_code }}`. I would recommend use using `$.queryPanel()` instead.

Do you mind giving a manual test to the latest version of this dashboard? It's quite difficult to spot all these nuances just looking at the jsonnet.

Reply:
> Do you mind giving a manual test to the latest version of this dashboard? It's quite difficult to spot all these nuances just looking at the jsonnet.

Sure, this is the current version:

<img width="1513" alt="Screen Shot 2022-10-11 at 19 12 28" src="https://user-images.githubusercontent.com/195371/195178772-f0fdae94-df5f-4714-afc6-d66f9ffb5aa1.png">

I'll try using the `$.queryPanel()` instead.

Reply:
I used a `$.queryPanel()` and I tried to make it look as similar as possible to a `$.qpsPanel()` by copying it's style:

<img width="1518" alt="Screen Shot 2022-10-11 at 19 42 13" src="https://user-images.githubusercontent.com/195371/195183992-b1378069-ee15-4812-9701-75772ea347f7.png">

This is with the series `error` selected:

<img width="516" alt="Screen Shot 2022-10-11 at 19 43 12" src="https://user-images.githubusercontent.com/195371/195184148-c67dd1a6-6f5e-4cb3-a1f9-f63cea15fc91.png">